renderer.py

```diff
@@ -94,14 +100,21 @@ def init_shader(self):
       vert_out.smooth("VEC3", "cc_ck_scale")
       vert_out.smooth("FLOAT", "cc_k4")
       vert_out.smooth("FLOAT", "cc_k5")
+      vert_out.smooth("VEC2", "primDepth")
       vert_out.smooth("VEC4", "uv")
       vert_out.no_perspective("VEC2", "posScreen")
       vert_out.flat("VEC4", "tileSize")
       vert_out.flat("INT", "flags")
+      vert_out.flat("INT", "geoMode")
+      vert_out.flat("INT", "othermodeL")
+      vert_out.flat("INT", "othermodeH")
 
       shader_info.push_constant("MAT4", "matMVP")
       shader_info.push_constant("MAT3", "matNorm")
       shader_info.push_constant("INT", "inFlags")
+      shader_info.push_constant("INT", "inGeoMode")
+      shader_info.push_constant("INT", "inOthermodeL")
+      shader_info.push_constant("INT", "inOthermodeH")
```

Comment:
Just a note:  push-constants are limited to 128bytes in total, blender will log a warning if it's above it.
But it should still be below that here
(the uniform buffers have a higher limitation that we probably never hit)